src/base/utils/version.h

```diff
@@ -186,10 +186,22 @@ namespace Utils
     };
 
     template <typename T, std::size_t N, std::size_t Mandatory>
-    inline bool operator!=(const Version<T, N, Mandatory> &left, const Version<T, N, Mandatory> &right)
+    constexpr bool operator!=(const Version<T, N, Mandatory> &left, const Version<T, N, Mandatory> &right)
     {
         return !(left == right);
     }
+
+    template <typename T, std::size_t N, std::size_t Mandatory>
+    constexpr bool operator<=(const Version<T, N, Mandatory> &left, const Version<T, N, Mandatory> &right)
+    {
+        return ((left < right) || (left == right));
```

Comment:
since I'm going to update PR once more time, would it be better writing:
```c++
return !(left > right);
```
?

Reply:
Yes, of course!